src/api.js

```diff
@@ -15,3 +15,11 @@ export const getReview = (review_id) => {
       return res.data;
     });
 };
+
+export const getComments = (review_id) => {
+  return axios
+    .get(`https://nc-be-games.onrender.com/api/reviews/${review_id}/comments`)
```

Comment:
You could create an axios instance with the base url which would make hitting the api easier from a dev perspective